apply/apply_Morris.py

```diff
@@ -65,20 +71,10 @@
     with open(cache_file, 'w') as fp:
         json.dump(partial_order, fp, indent=2)
 else:
-    file_exist = True
     with open(cache_file, 'r') as fp:
         partial_order = json.load(fp)
+
     for key, value in partial_order.items():
```

Comment:
Same comment as with apply_Sobol.py - the calculation appears to be conducted for no reason if the cache file exists.